index.html

```diff
@@ -321,6 +299,87 @@ <h1>Contact me</h1>
+            <div class="lorem">
+                <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Praesentium delectus iusto consequuntur perferendis aspernatur libero tenetur blanditiis facere, culpa quisquam, quasi officia dolorem consectetur. Modi, adipisci. Laborum maxime ipsa impedit ut, omnis libero sequi fuga at esse nostrum? Voluptatibus, labore! Corporis voluptatem cupiditate ab tempore iusto veritatis asperiores. Expedita possimus non qui vitae! Molestiae recusandae mollitia reprehenderit assumenda maiores dicta, adipisci fugiat eos illum dolorum vel eum maxime consequuntur facere repellendus odio incidunt excepturi </p>
+            </div>
+            <div>
+                <div class="categories">
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="categories">
+                    <ul>
+                        <li>github</li>
+                        <li>ruby</li>
+                        <li>Bootstraps</li>
+                    </ul>
+                </div>
+                <hr>
+                <div class="button" id="popupbutton">
+                    <button type="button" class="work-button1">See live<img src="img/21.svg" alt=""></button>
+                    <button type="button" class="work-button1">See source <img src="img/cat.png" alt="github"></button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+<!-- Make sure its in its own div -->
+
+<!-- Modal Pop Up Here -->
+
+       <div class="buttonmob" id="seeprojects">
+            
+          <div class="containerheader">
+            <h2>Tonic</h2>
+            <button class="close-button">&times;</button>
+          </div>
+          <div class="work-content work-content1">
+
+          <p>CANCOPY</p>
+          <ul>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          </div>
+          <img src="img/poppic.png" alt="poppc">
+         <p class="loremtextmob">Lorem ipsum dolor sit amet consectetur adipisicing elit. Autem excepturi cupiditate placeat suscipit quia  consequuntur reprehenderit sit fuga officiis cupiditate optio dolore labore vero itaque adipisci nihil ipsam, quisquam, quasi officia dolorem consectetur. Modi, adipisci. Laborum maxime ipsa impedit ut, omnis libero sequi fuga at esse nostrum? Voluptatibus, labore! Corporis quae recusandae, repudiandae !</p>
+<div class="categories">
+    <ul>
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+</div>
+<hr>
+<div class="button" id="popupbutton">
+    <button type="button" class="work-button1">See live <img src="img/21.svg" alt=""></button>
+    <button type="button" class="work-button1">See source <img src="img/cat.png" alt="github"></button>
+</div>
+<div class="active" id="overlay"></div>
+</div>
+<!-- Blues Baby -->
+<!--  -->
```

Comment:
- [ ] Kindly remove all the HTML code that you used for displaying each project in the projects section. The requirement of this project was to refactor your projects section by creating an array of projects in the javascript file and dynamically creating the HTML elements for each project with the data you add in the array. This way it will be easier for you in the future to add new projects to your portfolio by just adding the data of the project in this array.
 - 
<img width="993" alt="image" src="https://user-images.githubusercontent.com/61919617/201921121-871d2e72-d75f-4ff1-903c-4f0b0b831877.png">

 - [ ] Kindly create an array of projects in your Javascript file and create a function to populate the projects section with the data in that array. Example:
```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

- [ ] Please create a function in your `script.js` file to display the popup modal with the data of the project that the user clicks.